sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/models/GetManifestResult.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.containers.containerregistry.models;
+
+import com.azure.containers.containerregistry.implementation.ConstructorAccessors;
+import com.azure.core.util.BinaryData;
+
+/**
+ * The result from downloading a manifest from the registry.
```

Comment:
Since we are not using upload/download for manifests, we should update all docs to use set/get instead.